### ISSUE TITLE
History of mastectomy

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10211]
**Preferred term label:**
Mastectomy

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C0024881

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Mastectomy predisposes to Stewart-Treves syndrome 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000003
- label: History of mastectomy
- parents: HP:0032443
- pmid: 
- def: Past medical history of surgical breast removal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10211
- synonyms:
